db/migrations/20200414084022_add_storage_diff_checked_index.sql

```diff
@@ -0,0 +1,7 @@
+-- +goose Up
+create index storage_diff_checked_index on storage_diff(checked);
```

Comment:
nbd but I'd be a fan of including the schema in the table reference. also did you run `Reformat Code` on this file?